module-2/backend/week_four.md

```diff
@@ -10,14 +10,38 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+ A way to maintain state over HTTP request/response cycle.
+
 * What’s the difference between a session and a cookie?
+ A session is similiar to  a cookie except it allows you to have authorization/authentication.
```

Comment:
Could potentially do auth with a cookie, but it wouldn't be secure. A session is an encrypted cookie. A cookie by itself is not.